src/Features/CSharp/Portable/Completion/SuggestionMode/CSharpSuggestionModeCompletionProvider.cs

```diff
@@ -143,6 +125,13 @@ private bool IsLambdaExpression(SemanticModel semanticModel, int position, Synta
                 }
             }
 
+            // A lambda that is being typed may be parsed as a tuple without names 
```

Comment:
give an example.   i.e. "(a, b"  could be the start of a tuple/lambda, but "(a: b, c" could never be a lambda.